yt_idefix/fields.py

```diff
@@ -47,3 +62,266 @@ def setup_fluid_fields(self):
+                units=unit_system["temperature"],
+            )
+        else:
+
+            def _temperature(field, data):
+                return (
+                    data.ds.mu
+                    * (mh / kboltz)
+                    * (data[("gas", "pressure")] / data[("gas", "density")])
+                )
+
+            self.add_field(
+                ("gas", "Temperature"),
+                sampling_type="cell",
+                function=_temperature,
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                ("gas", "Temperature"),
+                units=unit_system["temperature"],
+            )
+
+        if (self.ds._dataset_type, "mach") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "mach"),
+                sampling_type="cell",
+                units="",
+            )
+            self.alias(
+                ("gas", "Mach"),
+                (self.ds._dataset_type, "mach"),
+                units="",
+            )
+            self.alias(
+                ("gas", "mach"),
+                (self.ds._dataset_type, "mach"),
+                units="",
+            )
+        else:
+
+            def _mach(field, data):
+                return np.sqrt(
+                    data[(self.ds._dataset_type, "vx1")] ** 2
+                    + data[(self.ds._dataset_type, "vx2")] ** 2
+                    + data[(self.ds._dataset_type, "vx3")] ** 2
+                ) / np.sqrt(
+                    data.ds.gamma
+                    * (data[("gas", "pressure")] / data[("gas", "density")])
+                )
+
+            self.add_field(
+                ("gas", "Mach"),
+                sampling_type="cell",
+                function=_mach,
+                units="",
+            )
+            self.alias(
+                ("gas", "mach"),
+                ("gas", "Mach"),
+                units="",
+            )
+
+        if (self.ds._dataset_type, "ndens") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "ndens"),
+                sampling_type="cell",
+                units=unit_system["length"] ** -3,
+            )
+            self.alias(
+                ("gas", "ndens"),
+                (self.ds._dataset_type, "ndens"),
+                units=unit_system["length"] ** -3,
+            )
+            self.alias(
+                ("gas", "Number Density"),
+                (self.ds._dataset_type, "ndens"),
+                units=unit_system["length"] ** -3,
+            )
+            self.alias(
+                ("gas", "number density"),
+                (self.ds._dataset_type, "ndens"),
+                units=unit_system["length"] ** -3,
+            )
+            self.alias(
+                ("gas", "Number density"),
+                (self.ds._dataset_type, "ndens"),
+                units=unit_system["length"] ** -3,
+            )
```

Comment:
please avoid defining multiple aliases per field. Each field should have at most 2 aliases:
`(self.ds._dataset_type, <pluto-standard-name>)` and `("gas", <yt-standard-name>)`

Reply:
Noted and changed accordingly!